Tools/jit/_targets.py

```diff
@@ -113,7 +112,7 @@ async def _compile(
         self, opname: str, c: pathlib.Path, tempdir: pathlib.Path
     ) -> _stencils.StencilGroup:
         # "Compile" the trampoline to an empty stencil group if it's not needed:
-        if opname == "trampoline" and not self.ghccc:
+        if opname == "trampoline":
             return _stencils.StencilGroup()
```

Comment:
These three lines can be removed. They were originally there so users didn't "pay" for the trampoline if `ghccc` wasn't supported for their platform. But now with this change it just unconditionally replaces the trampoline with nothing... which is wrong, haha. Commenting these lines out fixes the crashes!